src/test/java/com/puppycrawl/tools/checkstyle/internal/AllTestsTest.java

```diff
@@ -249,7 +251,8 @@ private static boolean shouldSkipInputFileNameCheck(String path, String fileName
                 // special directory for files that can't be renamed or are secondary inputs
                 || path.contains(File.separatorChar + "inputs" + File.separatorChar)
                 // all inputs must start with 'messages'
-                || path.contains(File.separatorChar + "translation" + File.separatorChar);
+                || path.contains(File.separatorChar + "translation" + File.separatorChar)
+                || fileName.charAt(0) == '.';
```

Comment:
I removed this change ... 
Use should not place any files in repo, we can come back to this if become very annoying.

Reply:
It becomes auto generated in each folder for mac os